at_commons/lib/src/keystore/at_key.dart

```diff
@@ -180,3 +277,9 @@ class AtValue {
     return 'AtValue{value: $value, metadata: $metadata}';
   }
 }
+
+/// Enumeration that represents value type in @protocol
+///
+/// binary - Represents binary data like images
+/// text   - Anything that's not binary falls into this category
+enum ValueType { binary, text }
```

Comment:
should we have json as well? so called will know when to json decode

Reply:
The valueType is removed.